.github/workflows/push_tests.yml

```diff
@@ -37,14 +38,9 @@ jobs:
           make lint-server
       - name: Lint src with eslint
         working-directory: ./client
-        if: github.event_name	!= 'pull_request'
         run: |
           make lint
-      # - name: Lint diff with eslint
-      #   working-directory: ./client
-      #   if: github.event_name	== 'pull_request'
-      #   run: |
-      #     make lint-diff
```

Comment:
If the lint tests are now run on all source, we can remove the `lint-diff` make commands, true?

Reply:
True, I'll add that in. Thanks!